src/Kiota.Builder/Extensions/StringExtensions.cs

```diff
@@ -112,5 +112,22 @@ public static string CleanupSymbolName(this string original, params string[] pre
 
             return original;
         }
+
+        /// <summary>
+        /// Cleanup the XML string
+        /// </summary>
+        /// <param name="original">The original string</param>
+        /// <returns></returns>
+        public static string CleanupXMLString(this string original)
+        {
+            if (string.IsNullOrEmpty(original))
+                return original;
+
+            return original.Replace("&", "&amp;")
```

Comment:
could we use this instead? https://docs.microsoft.com/en-us/dotnet/api/system.security.securityelement.escape?redirectedfrom=MSDN&view=net-6.0

Reply:
Agreed. That's much simpler.